environment_lsf.yml

```diff
@@ -0,0 +1,44 @@
+# You can use this file to create a conda environment for this pipeline:
+#   conda env create -f environment_lsf.yml
+
+dependencies:
+  - scipy=1.4.1=py38habc2bb6_0
+  - screed=0.9=py_1
+  - send2trash=1.5.0=py38_0
+  - seqtk=1.3=2
+  - setuptools=46.2.0=py38_0
+  - simplejson=3.17.0=py38h7b6447c_0
+  - six=1.14.0=py38_0
+  - ska=1.0=0
+  - snowballstemmer=2.0.0=py_0
+  - spectra=0.0.11=py_1
+  - sphinx=2.3.1=py_0
+  - sphinxcontrib-applehelp=1.0.2=py_0
+  - sphinxcontrib-devhelp=1.0.2=py_0
+  - sphinxcontrib-htmlhelp=1.0.3=py_0
+  - sphinxcontrib-jsmath=1.0.1=py_0
+  - sphinxcontrib-qthelp=1.0.3=py_0
+  - sphinxcontrib-serializinghtml=1.1.4=py_0
+  - sqlite=3.31.1=hbc83047_1
+  - terminado=0.8.3=py38_0
+  - testpath=0.4.4=py_0
+  - tk=8.6.8=hbc83047_0
+  - tornado=6.0.4=py38h7b6447c_1
+  - tqdm=4.43.0=py_0
+  - traitlets=4.3.3=py38_0
+  - urllib3=1.25.8=py38_0
+  - wcwidth=0.1.9=py_0
+  - webencodings=0.5.1=py38_1
+  - wheel=0.34.2=py38_0
+  - widgetsnbextension=3.5.1=py38_0
+  - xz=5.2.5=h7b6447c_0
+  - yaml=0.1.7=h1bed415_2
+  - zeromq=4.2.5=hf484d3e_0
+  - zipp=3.1.0=py_0
+  - zlib=1.2.11=h3c3a69f_1
+  - pip:
+    - dill==0.3.1.1
```

Comment:
there should be bam2fasta, sencha, sourmash installations too?

Reply:
pip installing would give errors with regards to numpy. we are going to have a call with Ruzhu from IBM to solve those errors and see how the pip packages can be installed on lsf